client/store/event.js

```diff
@@ -158,7 +158,7 @@ export default function(state = defaultEvents, action) {
     case RESET_ROUNDS: {
       return {
         byId: {...state.byId},
-        round: 1
+        round: 2
```

Comment:
I guess I don't understand how this works ? It feels fragile especially since we should be saving this stuff in the DB.

Reply:
It's a temporary fix while I get modify the Event model